.drone.yml

```diff
@@ -0,0 +1,49 @@
+kind: pipeline
+name: default
+
+services:
+- name: docker
+  image: docker:dind
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
+
+steps:
+- name: conform
+  image: autonomy/conform:c539351
```

Comment:
do we want to run a specific tag here or latest / always pull?

Reply:
specific tag for now. at least until we reach another semi-stable version of conform.